doc/whats_new/v0.23.rst

```diff
@@ -366,6 +366,10 @@ Changelog
   normalizing the vectors. :pr:`16632` by
   :user:`Maura Pintor <Maupin1991>` and :user:`Battista Biggio <bbiggio>`.
 
+- |Fix| :func:`preprocessing._label._encode` now `OneHotEncoder` and
```

Comment:
We don't usually put in a change log entry when we have merely improved an error message

Reply:
Got it!